[Bounty] mrat

ynot01 opened the pull request:
# Github documenting your Pull Request

Request Mentor Rat button
polls all mentors that are ghosts
selects from those who said yes
makes imaginary friend who is a rat and can leave when they choose
their name is their ckey
https://forums.yogstation.net/threads/15-mentor-mouse-aka-mrat-ss13-server.22889/

![image](https://i.imgur.com/sMFfyrJ.png)
![image](https://i.imgur.com/W3pQiX5.png)
![image](https://i.imgur.com/jbpIk2J.png)
![image](https://i.imgur.com/UdxUYTq.png)
![image](https://i.imgur.com/BkAdkOr.png)

## user side
![image](https://user-images.githubusercontent.com/28408322/126774628-ac508126-400a-4e92-b976-5cce5560c739.png)
![image](https://i.imgur.com/Mt9AjSc.png)
![image](https://i.imgur.com/2OkA2G8.png)


## mentor side
![image](https://i.imgur.com/GNMzFgY.png)
![image](https://i.imgur.com/ZWwQpMM.png)
![image](https://i.imgur.com/CYpeY2W.png)
![image](https://i.imgur.com/zgJkiEh.png)

# Wiki Documentation

Request Mentor Rat button in Mentor tab

# Changelog

:cl:  
rscadd: mrat  
/:cl:

ynot01 committed:
initial commit

ynot01 committed:
fixes a whole buncha things

ynot01 committed:
changes proc to verb

Bibby0110 reviewed (commented):
Can you verify that imaginary friends can point?

Might also want to check if runechat works, might be a little confusing for a new player if that’s that’s what they’re used to

ynot01 committed:
fixes a bunch of other things

TESTED AND WORKING

ynot01 committed:
humans only

ynot01 committed:
more things see desc

adjusted verb name & other things referencing the rat, let the rat pseudo-point, made the rat squeak whenever it talks, added a check for if someone already has an mrat

ynot01 committed:
mcolor

ynot01 committed:
radial appearance options, more adjustments
